README.md

```diff
@@ -16,8 +16,28 @@ This guide aims to improve the way your team uses [jQuery](http://jquery.com/).
 ## Table of Contents
 
 * [About jQuery](#about-jquery)
+* [Cache jQuery lookups](#cache-jquery-lookups)
 
 
 ## About jQuery
 
 [jQuery](http://jquery.com/) is a utility library for easy DOM access & manipulation, event handling, Ajax and more. By using jQuery you can write consise and expressive code which works across modern and legacy browsers. jQuery has extensive tests, detailed documentation, a large active community and an ecosystem of plugins.
+
+## Cache jQuery lookups
```

Comment:
Add summary / intro before "Why?" to explain what "caching jQuery lookups" means?